src/eduid_idp/idp_actions.py

```diff
@@ -71,7 +73,7 @@ def check_for_pending_actions(idp_app, user, ticket, sso_session):
     # Add any actions that may depend on the login data
     add_idp_initiated_actions(idp_app, user, ticket)
 
-    actions = idp_app.actions_db.get_actions(userid = user.user_id, session = ticket.key)
+    actions = idp_app.actions_db.get_actions(user.eppn, session = ticket.key)
```

Comment:
Wouldn't it make more sense to pass the user to get_actions and let it decide internally whether to use userid or eppn?

Reply:
The way backwards compatibility is implemented at the moment makes use of the different form of userid and eppn to decide how to use them - so if we provide the full user we would need another arg to distinguish both cases